### ISSUE TITLE
[Mac] Enable fullscreen button and "View" menu (enhancement)

### ISSUE THREAD

**Issue by jeanlain** [ValveSoftware/Dota-2#181]
For better integration in OS X, it would be nice to enable fullscreen mode via the "View" menu (in the menu bar) and via the fullscreen button (which would be the green button in 10.10+)
Right now, the menu is greyed out and the button does nothing. 
I recall that at some point, dota2 supported the fullscreen button, but no longer.

**Comment by gdrewb-valve** [comment]
Marking as an enhancement.

I don't believe Reborn sets a menu, at least I don't see one when I run Reborn on Mac.

**Comment by jeanlain** [comment]
I see in the menu bar "window" and "view" items. The "view" items has an "enter fullscreen" item that is disabled.

**Comment by gdrewb-valve** [comment]
The menu bar has Dota 2 on it?  I don't get any menu bar from Reborn, it's just whatever was active before starting Reborn.

**Comment by jeanlain** [comment]
Sure, every source game has a menu with its name, but it is not visible immediately at launch, due to another issue where the game isn't entirely brought to front when launched from steam. I could report that issue too. It happens in all source games. But switching to another app and back will activate the game's menu bar.

**Comment by gdrewb-valve** [comment]
I've checked into this and currently the lack of a fullscreen button is expected as Reborn does not use resizable windows.  This will stay open as an enhancement request for that.

I've logged the menu bar not appearing as SDL bug 3051 and it will be tracked over there.

**Comment by gdrewb-valve** [comment]
We heard back from the SDL team and apparently this has to do with starting from a shell script.  We can potentially change how Reborn is packaged to fix it.

**Comment by gdrewb-valve** [comment]
An upcoming update with put dota2 in a dota2.app bundle so that the menu will appear initially.  The menu function is unchanged.

**Comment by kisak-valve** [comment]
"- Fullscreen button will no longer be visible on the Watch Live screen when not applicable" in the [2017-05-04](http://store.steampowered.com/news/29270/) client update is related to this issue.
